Makefile

```diff
@@ -51,12 +51,23 @@ else ifeq ($(OS),Darwin)
 	@CGO_ENABLED=0 GOOS=linux go test -v -tags integration -c -o ./integration-test.dev $(PKG)
 	@docker run --name "integration-test" \
 		--network scylla_go_driver_public \
+		-v "$(PWD)/testdata:/testdata" \
```

Comment:
Nit: it would be preferable to fix this in a separate commit. The current commit is about the `docker-integration-test`, so this change seems unrelated.